package.json

```diff
@@ -19,11 +19,11 @@
   "homepage": "https://github.com/jgillich/npm-check-engines#readme",
   "dependencies": {
     "colors": "^1.4.0",
-    "npm": "^6.14.14",
+    "npm": "^8.1.3",
```

Comment:
Should `npm` be removed from `dependencies`? It's not imported or required. Dependencies are installed every time, and I'm not sure if a local installation would even be the npm CLI that gets used.

Reply:
Good point.

Reply:
The correct thing to do is to add `npm` to the package `engines` field, reflecting what versions support how the `npm` CLI is being used by this package.

https://github.com/jgillich/npm-check-engines/blob/3488f63234bc606c5a3a1d28f3867b3765c9dcd6/package.json#L26-L28

Reply:
Yeah I was literally just looking at doing that.